ui-core/src/index.ts

```diff
@@ -1,7 +1,15 @@
 import './styles/main.css';
 
 export { default as Button, ButtonProps } from './components/buttons/Button';
-export { default as Checkbox, CheckboxProps } from './components/inputs/checkbox';
+export { default as Checkbox, CheckboxProps } from './components/inputs/checkbox/Checkbox';
+export {
+  default as CheckboxList,
+  CheckboxListProps,
+} from './components/inputs/checkbox/CheckboxList';
+export {
+  default as CheckboxesTree,
+  CheckboxesTreeProps,
+} from './components/inputs/checkbox/CheckboxesTree';
```

Comment:
After having added an index file in components/inputs/Checkbox
```suggestion
export {
  Checkbox,
  CheckboxProps,
  CheckboxList,
  CheckboxListProps,
  CheckboxTree,
  CheckboxTreeProps,
} from './components/inputs/Checkbox';
```